src/karts/kart.cpp

```diff
@@ -2175,17 +2177,70 @@ void Kart::playCrashSFX(const Material* m, AbstractKart *k)
     {
         if (m_body->getLinearVelocity().length()> 0.555f)
         {
+			
+			float SpeedForMaxVolume = 15; //The speed at which the sound plays at maximum volume
+			float Volume;
+			float MaxVolume = 1; //The maximum volume a sound is played at 
+
+			if (k == NULL) //Collision with wall
+			{
+				Volume = (m_speed / SpeedForMaxVolume);
+			}
+			else
+			{
+				btVector3 ThisKartRotation = getVelocity();
+				btVector3 OtherKartRotation = k->getVelocity();
+				
+				
+				btScalar ScaledRotationDifference = ThisKartRotation.angle(OtherKartRotation) / 3.14159; //Difference in angles of karts in collision, Scaled by PI
+				
+				
+				btScalar SpeedDifference = getSpeed() - k->getSpeed();
```

Comment:
Don't use the bullet variables - just use plain float (and probably even Vec3 instead of btVector3)